content/en/admin/moderation.md

```diff
@@ -51,7 +53,7 @@ With this option active, no files from the server will be processed locally. Tha
 
 ### Limit {#limit-server}
 
-Equivalent to [limiting](#limit-user) all past and future accounts from the server. Previously known as "silencing".
+Equivalent to [limiting](#limit-user) all past and future accounts from the server. Previously known as "silencing". Notably, this will automatically turn all follows from the limited server into follow requests, even if the followed account it not "Locked".
```

Comment:
I'd replace `it not "Locked"` with `does not require approving followers`

Reply:
@abackstrom are you able review these changes?

Reply:
Thanks, this is now updated.